internal/tools/go.mod

```diff
@@ -56,7 +56,7 @@ require (
 	github.com/chavacava/garif v0.1.0 // indirect
 	github.com/chigopher/pathlib v0.19.1 // indirect
 	github.com/ckaznocha/intrange v0.1.2 // indirect
-	github.com/cloudflare/circl v1.3.7 // indirect
+	github.com/cloudflare/circl v1.3.8 // indirect
```

Comment:
Might conflict with https://github.com/DataDog/datadog-agent/pull/32452 who's touching the same dependency, hopefully github does merge correctly both